client/styles/debug.css

```diff
@@ -71,14 +82,25 @@ a:active {
     border-bottom: solid 1px #004e7f;
     color: white;
     background: #005a93;
+    padding: 15px;
+}
+
+.header.info.error {
+    margin: 15px -15px 15px -33px;
```

Comment:
Ah, I see why this was added - to tidy up fatal errors during a `dev/build`. I wouldn’t worry about that scenario too much as we’re currently outputting another `<html><body><link rel="stylesheet"...` in the middle of the build log when that happens, so that will need to be addressed properly at some point 😃